src/controllers/insertIssueTable.gs

```diff
@@ -0,0 +1,268 @@
+
```

Comment:
This is a pretty big file - are there separate classes / functions hiding in here - a large file is usually a smell of where code could be broken down into separate classes/modules. (Not critical - but its a big one to read)

Reply:
Hmm its actually not that big. Its a controller with more than just a single menu and single callback function as well as the wrapper definitions and the triggers. Not sure if it makes any sense to break that apart. On the other side, what means big? For what i deal with daily, this is a pretty small file  :)

Is there code, which can be optimized? Surely.

Reply:
Hey it's your code, appreciate you may be working with worst stuff at work though. 
worth a read:-
https://refactoring.guru/smells/long-method  
Smaller methods make your code more readable (code is read more than written ), easier to test and easier to refactor and maintain.
Nothing to block release.

Reply:
Your feedback is aways appriciated, good one and critica one the same way.
Still, are you referring to something specific in that file? I mean, ~250 lines of code is not much at all. Its not even half of the size of ie: models/jira/IssueFields.gs  :)

Reply:
Its the size of the methods as opposed to the file (which wasnt clear in my comment). The clean code book recommends 10 lines per method, honestly I find that unachievable but I "aspire" to it.
Fair point about "IssueFields" whoever coded that sucks ;)
there are still long methods in IssueFields - most of which are left over from the original code I moved in there - I work on it bit by bit.